### DISCUSSION TITLE
v0.11.0

### DISCUSSION THREAD

**Discussion by RobertCraigie** [RobertCraigie/prisma-client-py#833] [General]
## What's Changed

This release bumps the internal Prisma version from `4.15.2` to `5.4.2` bringing major performance improvements.

This release also adds support for retrieving metrics, e.g.

```py
from prisma import Prisma

client = Prisma()

metrics = client.get_metrics()
print(metrics.counters[0])
```

See [the docs](https://prisma-client-py.readthedocs.io/en/stable/reference/metrics/) for more information.

## Misc changes

- [Improved error message when nodeenv fails](https://github.com/RobertCraigie/prisma-client-py/pull/794)

![sponsors](https://github.com/RobertCraigie/prisma-client-py/assets/23125036/d5d8a53a-db22-4cc4-9937-e8eebc75bfd6)

<hr /><em>This discussion was created from the release <a href='https://github.com/RobertCraigie/prisma-client-py/releases/tag/v0.11.0'>v0.11.0</a>.</em>

**Comment by Insighttful** [comment]
@RobertCraigie Great port BTW!

It seems that Prisma actual resolved some issues with memory allocation which I am experiencing in Python-Prisma. A bump to the newest version of underlying Prisma should pick up these fixes? 

https://github.com/prisma/prisma/issues/19743

**Reply by RobertCraigie** [reply]
I should be able to publish a new release later today, in the meantime you can install from the github repo.
